assets/vendor/hypothesis-shared/dropdown-menu.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Note that everything in `assets/vendor/hypothesis-shared/` is a compiled asset built by the `update-hypothesis-shared.py` script